app/assets/stylesheets/pages/home/_subscribe.scss

```diff
@@ -0,0 +1,53 @@
+.subscribe{
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), image-url("bg-bg.jpeg");
+	width: 100%;
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-position: center;
+	margin-top: 92px;
+
+	&__wrap-item{
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		height: 100%;
+	}
+
+	&__item{
+		display: block;
+		text-align: center;
+		width: 875px;
+		padding: 134px 0;
+	}
+
+	&__title{
+		color: #fff;
+		font-size: 30px;
+		line-height: 2;
+		font-weight: normal;
+		margin: 0;
+	}
+
+	&__button{
+		margin-top: 40px;
+	}
+
+	&__button-link{
+		border: 2.5px solid #fff;
+		text-decoration: none;
+	    color: #fff;
+	    font-size: 30px;
+	    text-transform: uppercase;
+	    line-height: 1;
```

Comment:
this is useless, try make it with padding

Reply:
already  deleted this code, and i reduce padding top